content/en/blog/2023/egress-sni/index.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "[WIP] Routing HTTPS/TLS traffic to arbitrary remote destinations"
+description: "A generic approach to set up egress gateways and use them to route traffic to a restricted set of target remote hosts dynamically, supporting wildcard domains."
+publishdate: 2023-12-10
+attribution: "Gergő Huszty (IBM)"
+keywords: [traffic-management,gateway,mesh,mtls,egress,remote]
+---
+
+If you are using Istio to handle application-originated traffic to destinations outside of the mesh, you're probably familiar with the concept of egress gateways.
+Egress gateways can be used to monitor and forward traffic from mesh-internal applications to locations outside of the mesh.
+This is a useful feature if your system is operating in a restricted
+environment and you want to control what can be reached on the public internet from your mesh.
+
+## Background
+
+The use-case of configuring an egress gateway to handle arbitrary wildcard domains had been included in the [official Istio docs](/docs/tasks/traffic-management/egress/wildcard-egress-hosts/#wildcard-configuration-for-arbitrary-domains) up until version 1.13, but was subsequently removed because the documented solution was not officially supported or recommended and was subject to breakage in future versions of Istio.
```

Comment:
I think it's better to link to the old doc here:
```suggestion
The use-case of configuring an egress gateway to handle arbitrary wildcard domains had been included in the [official Istio docs](https://archive.istio.io/v1.13/docs/tasks/traffic-management/egress/wildcard-egress-hosts/#wildcard-configuration-for-arbitrary-domains) up until version 1.13, but was subsequently removed because the documented solution was not officially supported or recommended and was subject to breakage in future versions of Istio.
```

Reply:
I see! Sorry I removed the beginning as the linter complained about the full URL. It insisted to shorten it. Will try to figure out something that passes the linter...